fix broken test vllm:test_kernels - test_attention_selector.py::test_flash_attn

tracelogfb opened the pull request:
Summary: xformers calls torch.cuda.get_device_capability("cuda"). Our monkey patching version of get_device_capability didn't accept string argument.

Differential Revision: D74440549

houseroad reviewed (approved):
Looks good. Thanks for the fix.

houseroad commented:
Could you fix the lint issue?

houseroad commented:
DCO failure can be ignored, but pre-commit is something we need to fix :-)

DarkLight1337 commented:
Can you merge from main to fix pre-commit?

Commit:
fix broken test vllm:test_kernels - test_attention_selector.py::test_flash_attn (#17873)

Summary:
Pull Request resolved: https://github.com/vllm-project/vllm/pull/17873

xformers calls torch.cuda.get_device_capability("cuda"). Our monkey patching version of get_device_capability didn't accept string argument.

Reviewed By: houseroad

Differential Revision: D74440549
